### ISSUE TITLE
[feature request] EXIF view/editor

### ISSUE THREAD

**Issue by bouscram** [Polarrco/polarr_mac_support#25]
Having a complete view of the EXIF tags would be nice (its good to have some in the histogram view, but it is quite limited for the moment). And  even better an EXIF editor...

**Comment by skrimaging** [comment]
+1 for this! would be a great addition to have a full Exif editor built into Polarr!

**Comment by kopaka7** [comment]
Yeah, not having the camera model or the date taken can be a bit annoying.

**Comment by kopaka7** [comment]
I would like to be able to edit the time taken of a picture, because this can be an issue with multiple cameras and the most convenient thing would be to change this before uploading to a sharing service.

**Comment by seanpkent** [comment]
We now have a built in Exif viewer. You can check it out in the histogram (or cmd + i) 
We're still thinking about an editor, but for now I put in some basic editable metadata fields in the watermark tool (author, copywrite, and description)
@bouscram @skrimaging @kopaka7

**Comment by zhang13music** [comment]
+1 for this
Plus the bug of twisting the original Exif information when exporting really bothers me a lot. All my camera model or the date taken information is missing after editing and exporting. 
Urging a fix of this bug!

**Comment by seanpkent** [comment]
@zhang13music The bug of twisting Exif info has been fixed. You should see an update soon!
